src/components/Pools/V3/Stake/index.tsx

```diff
@@ -0,0 +1,391 @@
+/* eslint-disable max-lines */
```

Comment:
may be we can rename `Pools/V3` folder to `Pools/PangoChef` ? Whats your thoughts ?

Reply:
I put it as V3 to be consistent with the versions, but as we are going to change this, I can change the name of the folder